packages/site/src/components/PropsList.tsx

```diff
@@ -59,16 +58,39 @@ export const PropsList = ({
                 component: ReactNode;
                 required: boolean;
               }) => (
-                <Grid gap>
-                  <Grid.Cell size={{ xs: 2 }}>
-                    <div style={{ display: "flex", alignItems: "flex" }}>
-                      {item.key}
-                    </div>
-                  </Grid.Cell>
-                  <Grid.Cell size={{ xs: 1 }}>{item.required}</Grid.Cell>
-                  <Grid.Cell size={{ xs: 3 }}>{item.description}</Grid.Cell>
-                  <Grid.Cell size={{ xs: 5 }}>{item.component}</Grid.Cell>
-                </Grid>
+              <Grid>
+              <Grid.Cell size={{ xs: 2 }}>
+                <div style={{display: "flex", gap: "var(--space-smaller", flexWrap: "wrap"}}>
+                  {item.key}
+                  {item.required && (
+                    <InlineLabel>Required</InlineLabel>
+                  )}
+                </div>
+              </Grid.Cell>
+              <Grid.Cell size={{ xs: 3 }}>{item.component}</Grid.Cell>
+              <Grid.Cell size={{ xs: 7 }}>{item.description}</Grid.Cell>
+              </Grid>
+              )}
+            </DataList.Layout>
+            <DataList.Layout size="xs">
```

Comment:
Perfunctory small-viewport layout
![image](https://github.com/user-attachments/assets/9114fbef-1fb9-4513-a976-43181d2f9cb6)

Reply:
You can add `headerVisibility={{ xs: false, md: true }}` to the `DataList` parent.

<img width="467" alt="Screenshot 2024-12-03 at 5 02 16 PM" src="https://github.com/user-attachments/assets/2458cbb0-1f9f-4dfa-a5d0-adb12868d025">